src/main/java/io/aiven/kafka/connect/s3/config/AwsStsRole.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.s3.config;
+
+import java.util.Objects;
+
+final class AwsStsRole {
+
+    // AssumeRole request limit details here:
+    // https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html
+    public static final int MIN_SESSION_DURATION = 900;
```

Comment:
AWS lib has a dedicated constant for min session duration: `STSAssumeRoleSessionCredentialsProvider.DEFAULT_DURATION_SECONDS`

Reply:
This one hasa bit misleading name. Default is 3600: "By default, the value is set to 3600 seconds."
https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html -> DurationSeconds
That is why I prefer to both MIN and MAX explicitly. (I added MAX_SESSION_DURATION in a previous PR)

Reply:
Lets use our constant as an alias for AWS one:  `MIN_SESSION_DURATION = STSAssumeRoleSessionCredentialsProvider.DEFAULT_DURATION_SECONDS`